File: gym_be/src/Domain/Entities/Schedule.cs
using System;
using System.Collections.Generic;

namespace BackEnd.Domain.Entities;

public partial class Schedule
{
    public Guid Scheduleid { get; set; }

    public string? Dayofweek { get; set; }

    public int? Maxparticipants { get; set; }

    public DateTime? Starttime { get; set; }

    public DateTime? Endtime { get; set; }

    public virtual ICollection<Appointment> Appointments { get; set; } = new List<Appointment>();
}

File: gym_fe/src/type/schedule.ts
export interface Schedule {
  scheduleid: string;
  dayofweek: string;
  starttime: number;
  endtime: number;
  maxparticipants: number;
}

export interface ScheduleFormData {
  scheduleid: string;
  dayofweek: string;
  maxparticipants: number;
  starttime: number;
  endtime: number;
}

export interface ScheduleApiResponse {
  items: Schedule[];
  totalPages: number;
  currentPage: number;
  totalItems: number;
}